Resources/Locale/en-US/_DV/prototypes/catalog/fills/crates/furniture-crates.ftl

```diff
@@ -0,0 +1,4 @@
+ent-CrateFurnitureDecorativePlants = potted plants crate
+    .desc = Spruce up your place with a little foliage. Contains four pre-potted plants.
+ent-CrateFurnitureDecorativePlantsPlastic = potted plastic plants crate
+    .desc = Spruce up your place with a little foliage. Contains four pre-potted plastic plants.
```

Comment:
dont need this since its not changing an upstream one
use name: and description: like normal